deployment/GI05MOAS-PG515_Deploy.csv

```diff
@@ -0,0 +1,8 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
+AR35-05,,,,GI05MOAS-PG515-00-ENG000000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGCON-GLDRGP-00515,,,,200,200,
+AR35-05,,,,GI05MOAS-PG515-01-CTDGVM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-CTDGVM-09231,,,,200,200,
+AR35-05,,,,GI05MOAS-PG515-02-DOSTAM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-DOSTAM-00414,,,,200,200,
+AR35-05,,,,GI05MOAS-PG515-03-FLORTM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTM-03501,,,,200,200,
+AR35-05,,,,GI05MOAS-PG515-04-FLORTO000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTO-01239,,,,200,200,
+#AR35-05,,,,GI05MOAS-PG515-05-NUTNRM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-NUTNRM-00760,,,,200,200,need vendor cal doc to create csv
+AR35-05,,,,GI05MOAS-PG515-06-PARADM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-PARADM-50185,,,,200,200,
```

Comment:
Looks good.  Confirmed from 3304-00170-00033 (with exception of NUTNR as noted in comments).